edn-react/src/pages/Chat.tsx

```diff
@@ -10,6 +25,7 @@ export default function Chat() {
       </div>
       <div className="mx-auto max-w-2xl py-20 sm:py-20 lg:py-6">
         <div className="text-center">
+          <span>{renderElement(data)}</span>
```

Comment:
This just lives here to display the response from openAI for the time being. This will eventually integrate with the chat window. I just wanted something in the UI to show that the response works.